website/docs/compose/podman-compose.md

```diff
@@ -1,23 +1,31 @@
 ---
 title: Podman Compose with Podman Desktop
-sidebar_position: 2
+sidebar_position: 3
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ## Introduction
 
-[Podman Compose](https://github.com/containers/podman-compose#readme) is a community-driven tool that allows developers to use the [Compose spec](https://compose-spec.io) to run many containers at the same time. Podman Compose requires a [Compose YAML](https://compose-spec.io/) file with definitions about the containers that need to communicate.
+The [Compose spec](https://compose-spec.io) is a container specification that allows you to define a multi-container application in a simple YAML file.
 
 If you have not used a [Compose file](https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file) or the [Compose spec](https://compose-spec.io/), visit [this guide](https://github.com/compose-spec/compose-spec/blob/master/spec.md) for more information.
 
+### Implementations
+
+There are two implementations that Podman Desktop supports:
+
+- [Podman Compose](https://github.com/containers/podman-compose): The Python implementation of the Compose specification with native Podman integration.
+- [Compose](https://github.com/docker/compose): The Go reference implementation of the Compose specification.
```

Comment:
If the Go reference implementation becomes also our preferred implementation, consider enumarating it first (+ remove fluff):

```suggestion
- [Compose](https://github.com/docker/compose): The Go reference implementation.
- [Podman Compose](https://github.com/containers/podman-compose): Alternative Python implementation with native Podman integration.
```